src/Administration/Resources/app/administration/src/module/sw-settings-tax/component/sw-tax-rule-card/index.js

```diff
@@ -135,11 +134,10 @@ export default {
         getList() {
             this.taxRulesLoading = true;
 
-            return this.taxRuleRepository.search(this.taxRuleCriteria, Context.api).then((response) => {
+            this.taxRuleRepository.search(this.taxRuleCriteria).then((response) => {
```

Comment:
Removing the result as a return value would be breaking change. Because `getList` now returns `undefined`, previously a Promise

Reply:
@jleifeld okay, I will bring back the removal :-) (and add a deprecation?)

Is this also the case for the `onConfirmDelete()` or is that in that sense fine, as it is only a handler method?

Reply:
In both cases the return value needs to be the same. But I wouldn't add a deprecation to it because it makes sense that it returns a promise. Otherwise if you need to wait for it (like when someone overrides it with a parent call), you can't wait until it is resolved

Reply:
FYI: I added the return values.